src/actions/index.js

```diff
@@ -148,7 +166,12 @@ export function urlForState(dongleId, log_id, start, end, prime) {
 }
 
 function updateTimeline(state, dispatch, log_id, start, end, allowPathChange) {
-  dispatch(checkRoutesData());
```

Comment:
Added in https://github.com/commaai/connect/pull/419, and I'm not sure why. It ends up re-querying the API twice on loading any route (from URL only). This ends up breaking something in `ACTION_ROUTES_METADATA` as it is also called twice, with the second time having set `start` and `end` values for the `segmentRange`, but this all only works if it's `NaN` on load (which is what master does because the first query has `limit` set to 0 lol

In summary: connect barely works as-is